h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -56,6 +60,11 @@
   private final boolean convertUnknownCategoricalLevelsToNa;
   private final boolean convertInvalidNumbersToNa;
   private final boolean useExtendedOutput;
+  private boolean getLeafAssignment=false;
```

Comment:
we should have this field final and use the Builder pattern that is used for the rest of the configuration properties, the config object should have the setters and the field should be final. What do you think?

Reply:
also in the constructor of EasyPredictModelWrapper we can check that leafnode assignment is only enabled when the users uses SharedTreeModel

otherwise they will fail at predict time on cast exception